dmutils/email/dm_notify.py

```diff
@@ -18,11 +22,18 @@ def __init__(
             self,
             govuk_notify_api_key,
             govuk_notify_base_url='https://api.notifications.service.gov.uk',
-            logger=None
+            logger=None,
+            replacement_email_address=FROM_CURRENT_APP,
     ):
-        """Set up logging and mail client."""
+        # Set up logging and mail client.
         self.logger = logger or current_app.logger
         self.client = self._client_class(govuk_notify_api_key, govuk_notify_base_url)
+        # determine replacement_email_address
```

Comment:
This is a bit of a non-comment :trollface:

Reply:
Agreed, but was just trying to fit in with the above non-comment.